python/tk_multi_loader/model_entity.py

```diff
@@ -43,16 +43,30 @@ def __init__(self, parent, overlay_widget, entity_type, filters, hierarchy):
         self._entity_icons["Version"] = QtGui.QIcon(QtGui.QPixmap(":/res/icon_Version_dark.png"))
         
         ShotgunOverlayModel.__init__(self, 
-                                     parent, 
+                                     parent,
                                      overlay_widget, 
                                      download_thumbs=False, 
                                      schema_generation=4)
         fields=["image", "sg_status_list", "description"]
         self._load_data(entity_type, filters, hierarchy, fields)
+
+        self.reSelector = {'entity': None,
```

Comment:
Ticket #8253
After filling model from local shotgun cache dialog "presses" home button and selects current shot for example. Model checks if local cache hasn't diverged from actual shotgun data, if it is the case, items in model usually are destroyed and model is filled again. After that selection is not restored which means user was thrown out of selected shot and has to press home button again.